### PR TITLE
EZP-25969: Incorrect path length in suggested list

### PR DESCRIPTION
**Pull request by dew326**
**Jira ticket:** https://jira.ez.no/browse/EZP-25968

**Screenshot:**
paths-suggested-location

**TO DO:**
- [x] corrected path in suggested list
- [x] improved layout on smaller resolutions

### CODE REVIEW

**Comment by dew326** [comment]
ping @lserwatka @sunpietro

File: bundle/Resources/public/js/views/cof-contentcreationview.js [hunk 1]
```diff
@@ -435,23 +439,43 @@ YUI.add('cof-contentcreationview', function (Y) {
...
+                    longPath = pathSeparator,
+                    substringEnd;
 
                 if (locationPath.length) {
                     longPath = locationPath.reduce(function (path, location) {
                         return path + location.get('contentInfo').get('name') + pathSeparator;
                     }, '');
                 }
 
+                if (locationPath.length >= 3) {
```
Comment (lserwatka) [inline bundle/Resources/public/js/views/cof-contentcreationview.js]: Int 3 is used in several places, if this is something that might change, you could move this is to variable.

File: bundle/Resources/public/js/views/cof-contentcreationview.js [hunk 2]
```diff
@@ -435,23 +439,43 @@ YUI.add('cof-contentcreationview', function (Y) {
...
+                    shortPath = locationPath[0].get('contentInfo').get('name') + insidePathSeparator + locationPath[locationPath.length - 1].get('contentInfo').get('name') + pathSeparator;
```
Comment (sunpietro) [inline bundle/Resources/public/js/views/cof-contentcreationview.js]: Isn't it easier to read when it's written this way?

``` javascript
shortPath = [
    locationPath[0].get('contentInfo').get('name'),
    insidePathSeparator,
    locationPath[locationPath.length - 1].get('contentInfo').get('name'),
    pathSeparator
].join('');
```

**Comment by lserwatka** [comment]
+1

**Comment by lserwatka** [comment]
@dew326 please rebase

File: bundle/Resources/public/js/views/cof-contentcreationview.js [hunk 3]
```diff
@@ -440,23 +444,49 @@ YUI.add('cof-contentcreationview', function (Y) {
                 itemTemplate = this.get('suggestedItemTemplate'),
                 documentFragment = Y.one(document.createDocumentFragment()),
                 pathSeparator = '/',
+                endLongPath = '...',
+                insidePathSeparator = pathSeparator + endLongPath + pathSeparator,
+                maxPathLength = 40,
+                minVisibleLenth = 3,
```
Comment (sunpietro) [inline bundle/Resources/public/js/views/cof-contentcreationview.js]: `minVisibleCharsNumber`

**Comment by dew326** [comment]
ping @sunpietro

**Comment by sunpietro** [comment]
+1

**Pull request merged**